safe-global/safe-wallet-web
# Feat: Show warning for address poisoning transactions

## What it solves

Resolves #3832

## How this PR fixes it
Check for transactions that have been flagged as imitations of previous legitimate transactions. If they are flagged, display a warning on the tx in the history.

## How to test it
- Open a safe with malicious imitation transactions, (https://address_poisoning--walletweb.review.5afe.dev/transactions/history?safe=eth:0xf0ddc6435b47d161d46f51f2ea7c5f32f90b98be)
- See that the imitation transactions have the warning displayed, as in the designs:
https://www.figma.com/design/ptTs6lDBeUuLNySroJ5PiF/Web-Master-File?node-id=11151-49354&t=gfp1I4vZwEzFOo7i-1

## Screenshots
![image](https://github.com/safe-global/safe-wallet-web/assets/17801424/1189934e-0067-4370-a47f-47b61e04d1d6)

## Checklist
